l10n_br_base/models/res_country.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2009  Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class Country(models.Model):
```

Comment:
I don't see reason to change file formation, should just change import order:

https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_base/models/res_country.py

Reply:
Then eventually we may ask ourselves if that wouldn't be better to apply the black precommit hook in 12.0 as well (and eventually before) the 13.0 port. Indeed. 10.0 to 12.0 concentrated the most changes anyway as the Python version changes and ans 10.0 is mostly a cleaned 8.0 port while 12.0 is getting the major nfe/fiscal document refactor. 13.0 will have changes because of the account.invoice to account.move refactor. But given the l10n_br_fiscal module extraction this will only impact half of the localization modules and in minor ways (specially compared to the 10 to 12 changes). So I suggest using black in 12.0 already and extract later. We used that startegy in our repos like Shopinvader, but I believe this is also used in the OCA repos like  https://github.com/OCA/mis-builder/tree/12.0. What do you think?